hw0603uitask2/src/main/java/com/tashad16a/gmail/hw0603ui/uigooglepodcasts/ViewPagerAdapter.java

```diff
@@ -0,0 +1,51 @@
+package com.tashad16a.gmail.hw0603ui.uigooglepodcasts;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import com.tashad16a.gmail.hw0603_ui.R;
+import com.tashad16a.gmail.hw0603ui.uigooglepodcasts.fragment.PageFragment;
+import com.tashad16a.gmail.hw0603ui.uigooglepodcasts.fragment.SimplePageFragment;
+
+public class ViewPagerAdapter extends FragmentPagerAdapter {
+    private Context mContext;
+
+    public ViewPagerAdapter(Context context, FragmentManager fragmentManager) {
+        super(fragmentManager);
+
+        mContext = context;
+    }
+
+    @Override
+    public Fragment getItem(int position) {
+        PageFragment pageFragment = new PageFragment();
```

Comment:
issue is not resolved. Every time when getItem method called you create new instance of fragment. Please instead of that create it once, save to variable and reuse it